data/THammami01.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Tarek Hammami",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "A passionate student developer that loves competitive programming, and software that is both free and open-sourced.",
+  "avatar": "https://avatars.githubusercontent.com/u/50141415?v=4",
```

It's best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/THammami01.png",
```